released/packages/coq-library-fol/coq-library-fol.1.0+8.20/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+version: "1.0+8.20"
```

Comment:
```suggestion
```

Reply:
Ah, thanks for pointing out that this is obsolete.

Reply:
@JoJoDeveloping the idea here is that you should completely leave out any `version` field, because the version is already determined by the file path.

Reply:
So for precision it's "redundant" and not "obsolete".